README.md

```diff
@@ -1039,26 +1232,26 @@
 
     ```javascript
     // bad
-    Jedi.prototype.jump = function () {
+    Jedi.prototype.jump = function() {
```

Comment:
I thought you indicated you went through and fixed all these, but, I keep seeing them. May want to do a global find/replace for `function(` with `function (`